src/main/java/com/google/codeu/utilities/MessageUtil.java

```diff
@@ -8,14 +8,31 @@
 repeated code in the servlets and other pojo files (Plain Old Java Object files) :)
 */
 public class MessageUtil {
-
+  
+  /* Text formatting replacement for using Markdown inputs by the user.
+   * The following input turn into following outputs:
+   * 1) ## TEXT ## -> <b> TEXT </b>
```

Comment:
Heh, looks like Github did some automatic styling on my comment. I meant technically the "b" tags should be "strong" tags.